azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -406,7 +403,7 @@ private void getToken(final String userToProxy) throws InterruptedException,
   private void cancelHiveToken(final Token<? extends TokenIdentifier> t)
       throws HadoopSecurityManagerException {
     try {
-      final IMetaStoreClient hiveClient = this.hiveMetaStoreClientFactory.create();
+      final IMetaStoreClient hiveClient = new IMetaStoreClientFactory(new HiveConf()).create();
```

Comment:
It seems odd that cancelHiveToken is a local cluster operation. Do you know why thats the case?